.github/workflows/ci-build.yaml

```diff
@@ -401,7 +401,15 @@ jobs:
       - run: yarn --cwd ui test
       - run: yarn --cwd ui lint
       - run: yarn --cwd ui deduplicate
-      - run: git diff --exit-code
+      - if: github.actor == 'dependabot[bot]'
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+          git add ui/yarn.lock
+          git commit -m 'chore: update yarn.lock'
+          git push
+      - if: github.actor != 'dependabot[bot]'
+        run: git diff --exit-code
```

Comment:
this should still run for dependabot as well. it should have no diff after the commit

Reply:
I thought I should skip this because the current method doesn't trigger the actions again with a new commit made by the bot, but since the script is executed based on the last commit made by the bot, it should work like you said.